src/main/java/com/google/opensesame/servlets/MentorsServlet.java

```diff
@@ -21,47 +22,26 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<MentorObject> mentors = new ArrayList<MentorObject>();
+    ArrayList<UserData> mentors = new ArrayList<UserData>();
     ArrayList<String> projects = new ArrayList<String>();
     projects.add("OpenSesame");
 
     ArrayList<String> ObiSkills = new ArrayList<String>();
     ObiSkills.add("Meme god");
     ObiSkills.add("HTML wrangler");
-    MentorObject Obi =
-        new PersonBuilder()
-            .name("Obi")
-            .gitHubID("Obinnabii")
-            .description("Obi is awesome.")
-            .interestTags(ObiSkills)
-            .projectIDs(projects)
-            .buildMentor();
+    UserData Obi = null;
```

Comment:
Do we really want to add a null UserData? Should we just remove the `Obi`, `Sami`, and `Richie` objects? 😅

Reply:
Seconded. We should be good to remove these!